<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: #EFF4D1;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>47</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #1D1E4E;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: 800;
font-size: 50px;
line-height: 136px;

color: #000000;</string>
    </property>
    <property name="text">
     <string>Laporan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: 800;
font-size: 50px;
line-height: 136px;

color: #000000;</string>
    </property>
    <property name="text">
     <string>Pemesanan</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelPemesanan">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>190</y>
      <width>158</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-weight:bold;
background-color: #EFF4D1;
border: 0px solid #FFFFFF;
font-size:15px;
color: #000000;</string>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tanggal</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>26/11/2021</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>190</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-weight:bold;
background-color:white;
font-size:15px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Paket</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>190</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
background-color:white;
font-weight:bold;
font-size:15px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Status Bayar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>190</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-weight:bold;
background-color:white;
font-size:15px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Status CV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>122</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFCE31;
border-radius: 20px;
font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 14px;
line-height: 19px;
color: #1D1E4E;
</string>
    </property>
    <property name="text">
     <string>Go Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detailPaket">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>222</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 41px;
/* identical to box height */

text-decoration-line: underline;

color: #428BCA;</string>
    </property>
    <property name="text">
     <string>Detail</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="detailBayar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>222</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 41px;
/* identical to box height */

text-decoration-line: underline;

color: #428BCA;</string>
    </property>
    <property name="text">
     <string>Detail</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="detailCV">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>222</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Open Sans;
font-style: italic;
font-weight: normal;
font-size: 15px;
line-height: 41px;
/* identical to box height */

text-decoration-line: underline;

color: #428BCA;</string>
    </property>
    <property name="text">
     <string>Detail</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
